package.json

```
@@ -16,7 +16,7 @@
     "postinstall": "husky install",
     "prepare": "node -e \"try { require('husky').install() } catch (e) {if (e.code !== 'MODULE_NOT_FOUND') throw e}\"",
     "test": "jest -c jest.config.ts",
-    "test:integration": "jest -c integration-tests/jest.integration.config.ts",
+    "test:integration": "jest -c tests/integration-tests/jest.integration.config.ts",
     "test:integration-dev": "jest --setupFiles '<rootDir>/.env' -c integration-tests/jest.integration.config.ts"
```

This line needs updating too. Can we also make the name consistent with the other repo?

```suggestion
    "test:integration:dev": "jest --setupFiles '<rootDir>/.env' -c tests/integration-tests/jest.integration.config.ts"
```